### PR TITLE
Added DSC CVR Lead details

### PR DESCRIPTION
**Pull request by Aineshphanithi**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import karthik from "../../photos/karthik.jpg";
 import MuhammadHassanRazaa from "../../photos/MuhammadHassanRazaa.jpg";
 import advaithunnikrishnan from "../../photos/advaithunnikrishnan.jpg";
 import AsishRaju from "../../photos/AsishRaju.jpg";
+import Aineshphanithi from "../../photos/Ainesh.jpg";
```
Comment (abhinavsri360) [inline src/components/json/worldData.js:12]: @Aineshphanithi 
This should be
import Aineshphanithi from "../../photos/Aineshphanithi.jpg";

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]

**Comment by aravindvnair99** [comment]
@Aineshphanithi Updating this PR also works instead of closing and sending new one.

**Comment by Aineshphanithi** [comment]
Fish, thanks for that man!

File: src/components/json/worldData.js (hunk 1 above)
Comment (abhinavsri360) [reply src/components/json/worldData.js:12]: Hey, just make this change on the main branch and you'll be gtg.

Comment (aravindvnair99) [reply src/components/json/worldData.js:12]:
```suggestion
import Aineshphanithi from "../../photos/Aineshphanithi.jpg";
```

**Review by aravindvnair99 — APPROVED** [review]

**Review by abhinavsri360 — APPROVED** [review]
I just saw @aravindvnair99 is everygreen on GitHub 😂

**Pull request merged**